packages/flutterfire_ui/lib/src/i10n/lang/hi.dart

```diff
@@ -0,0 +1,263 @@
+import '../default_localizations.dart';
+
+class EnLocalizations extends FlutterFireUILocalizationLabels {

```

Comment:
Hello Aditya, shouldn't it be **HiLocalizations**?

Reply:
Yes Sorry about that I'll fix this.